### PR TITLE
Bug 2019360: Create the Virtualization Overview Status card

### PR DESCRIPTION
**Pull request by pcbailey**
This PR adds the Status card to the Virtualization Overview.

Full page screenshot:
Selection_011

Card-only screenshot:
Selection_012

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/models/index.ts [hunk 1]
```diff
@@ -231,3 +231,16 @@ export const HyperConvergedModel = {
   crd: true,
   plural: 'hyperconvergeds',
 };
+
+export const NetworkAddonsConfigModel: K8sKind = {
+  label: 'Network Addons Config',
+  labelPlural: 'Network Addons Configs',
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/models/index.ts:237]:
```suggestion
  label: 'NetworkAddonsConfig',
  labelPlural: 'NetworkAddonsConfigs',
```
I think the new convention is to use the kind also as the label
@glekner do you know if we should use human readable or kind as labels ?

File: frontend/packages/kubevirt-plugin/src/components/virtualization-overview/VirtOverviewStatusCard.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardLink from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardLink';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
+import HealthItem from '@console/shared/src/components/dashboard/status-card/HealthItem';
+import { NetworkAddonsConfigModel } from '../../models';
+
+export const NetworkAddonsConfigResource: FirehoseResource = {
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/virtualization-overview/VirtOverviewStatusCard.tsx:35]: Do we have a fallback for users that can't access the resources ?
( did we plan what cards in the overview will be available for none-admins ? )

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/virtualization-overview/VirtOverviewStatusCard.tsx:35]: hmm, maybe not show the overview for none-admins ?

File: frontend/packages/kubevirt-plugin/src/models/index.ts (hunk 1 above)
Comment (glekner) [reply frontend/packages/kubevirt-plugin/src/models/index.ts:237]: yep, use kind as label AFAIK

**Pull request merged**